### PR TITLE
Issue #SB-16708 fix: Upload video and dismiss video pop up condtion handled

### PR DESCRIPTION
**Pull request by JagadishPujari**

### CODE REVIEW

File: src/app/client/src/app/modules/cbse-program/components/ckeditor-tool/ckeditor-tool.component.ts [hunk 1]
```diff
@@ -569,16 +570,29 @@ getAllVideos(offset, query) {
           const errInfo = { errorMsg: 'Video upload failed' };
           return throwError(this.cbseService.apiErrorHandling(err, errInfo));
         })).subscribe((response) => {
-          this.selectedVideo = response.result;
-          this.addVideoInEditor();
-          this.showVideoPicker = false;
-          this.showVideoUploadModal = false;
+          //Read upload video data
+          this.getUploadVide(response.result.node_id);
```
Comment (cksachdev) [inline src/app/client/src/app/modules/cbse-program/components/ckeditor-tool/ckeditor-tool.component.ts]: @JagadishPujari Correct the spelling.
cc: @vinukumar-vs

File: src/app/client/src/app/modules/cbse-program/components/mcq-creation/mcq-creation.component.ts [hunk 2]
```diff
@@ -174,20 +174,25 @@ export class McqCreationComponent implements OnInit, OnChanges, AfterViewInit {
...
-    this.showSolutionDropDown = false;
+    if(event){
+      this.videoShow = false;
+      this.videoSolutionData = event;
+      this.videoSolutionName = event.name;
+      this.solutionValue = event.identifier;
+      this.videoThumbnail = event.thumbnail;
+      const videoMedia: any = {};
+      videoMedia.id = event.identifier;
+      videoMedia.src = event.downloadUrl;
```
Comment (cksachdev) [inline src/app/client/src/app/modules/cbse-program/components/mcq-creation/mcq-creation.component.ts]: @JagadishPujari Check if there is any modification required in url before adding it to the theme object.

File: src/app/client/src/app/modules/cbse-program/components/ckeditor-tool/ckeditor-tool.component.ts (hunk 1 above)
Comment (JagadishPujari) [reply src/app/client/src/app/modules/cbse-program/components/ckeditor-tool/ckeditor-tool.component.ts]: Corrected spelling

File: src/app/client/src/app/modules/cbse-program/components/mcq-creation/mcq-creation.component.ts (hunk 2 above)
Comment (JagadishPujari) [reply src/app/client/src/app/modules/cbse-program/components/mcq-creation/mcq-creation.component.ts]: We are setting a relative path in manifest media array in ECML(theme object)

**Pull request closed without merging**